.github/workflows/release-update-cloudfoundry-index.yml

```
@@ -0,0 +1,37 @@
+# This action updates the CloudFoundry java-buildpack release index after each release.
+# See https://github.com/cloudfoundry/java-buildpack/blob/main/docs/extending-repositories.md
+# Prerequisite: the repo must have a branch named "cloudfoundry".
+
+name: Update cloudfoundry release index
+on:
+  release:
+    types: [released]
```

Try `published`:
```suggestion
    types: [published]
```

We're doing a similar after-release thing in the Splunk repo: https://github.com/signalfx/splunk-otel-java/blob/main/.github/workflows/publish-docker-image-ghcr.yml, and it works with the `published` type

---

I tried that first and had similar issues.  I'll go ahead and switch though.  The main difference is if it is triggered for pre-releases which I don't think we use, so it shouldn't matter.